lib/matplotlib/table.py

```diff
@@ -118,7 +135,7 @@ def _set_text_position(self, renderer):
         l, b, w, h = bbox.bounds
 
         # draw in center vertically
-        self._text.set_verticalalignment('center')
+        self._text.set_verticalalignment('center_baseline')
```

Comment:
Sorry for being unclear (and so late to respond!)

See https://github.com/matplotlib/matplotlib/blob/master/doc/api/api_changes/README.rst